tesseract-clickhouse/src/lib.rs

```diff
@@ -74,15 +74,15 @@ impl Backend for Clickhouse {
         Box::new(fut_stream)
     }
 
+    // https://users.rust-lang.org/t/solved-is-it-possible-to-clone-a-boxed-trait-object/1714/4
```

Comment:
maybe a silly question :) but I'm not sure why this method was moved?

Reply:
I kept getting a warning that the order of the methods in here was different from the trait definition in https://github.com/hwchen/tesseract/blob/master/tesseract-core/src/backend.rs. Just wanted to get rid of it 😄